### DISCUSSION TITLE
There is a bug in chia mining.

### DISCUSSION THREAD

**Discussion by YYTYYYYYY** [Chia-Network/chia-blockchain#10843] [Support]
xch1p4mhhlhgfukwpg8khyq933unaa2ju3u8l70fqga5mccv4mpx7wfs3fsy2e

I claimed 35 CHIAs from the self-pool yesterday to that address.
However, only the transaction was recorded and no coins were actually received.
please solve this problem
CHIA GUI version that claimed compensation was 1.30, and with version 1.29 today, 3.5 CHIA claims were successful.

**Comment by loppefaaret** [comment]
i see 20 transactions looking like claims of 35 XCH on that address on the 20th ?
you might want to update to 1.3.1 - and if there is still missing coins, let me know
